### DISCUSSION TITLE
WSL2 not work on Azure

### DISCUSSION THREAD

**Discussion by shuanliMS** [chocolatey/choco#2489] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/42368534/145735309-a58612e1-e095-4408-828c-5eeef37974ca.png)
But if we are using WSL --set-default-version 2 it will run successfully.
![image](https://user-images.githubusercontent.com/42368534/145735347-aa8f0f1c-172a-4e3b-8f00-2d2d24c60db5.png)
I managed to check chocolateyInstall.ps1 found below:
elseif ($packageArgs.Version -eq 2 -and
    $OSProductType -eq [OSType]::WorkStation -and
    (($OSMajorBuildNumber -in (18362, 18363) -and $OSMinorBuildNumber -ge 1049) -or
        ($OSMajorBuildNumber -ge 19041))) {
We get information from PG, we have enabled wsl2 for azure win 10 after this patch https://support.microsoft.com/en-gb/topic/november-9-2021-kb5007186-os-builds-19041-1348-19042-1348-and-19043-1348-033ee59c-e9b7-4eaf-8ee7-b3512bb1a0aa, please check if anyone can help fix this issue in Choco?

**Comment by TheCakeIsNaOH** [comment, marked as answer]
Hi,

This does not appear to be an issue with Choco itself, but with the `wsl2` package from the community repository.

I would suggest contacting the maintainer of the `wsl2` package, here are some options:
https://community.chocolatey.org/packages/wsl2#discussion
https://community.chocolatey.org/packages/wsl2/ContactOwners
https://github.com/daronmcintosh/chocolatey-packages/tree/master/wsl2

**Reply by shuanliMS** [reply]
thanks, let me ask in wsl2 channel
